Resources/public/templates/partials/tree.hbt

```diff
@@ -1,7 +1,7 @@
 <ul class="ez-tree-level">
 {{#each children}}
-    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}">
```

Comment:
what's the point of moving the title from the link to the list item ? I mean you can find the link and just retrieve the parent node to get the list item. I'm saying that because with this change, you'll end up with something like:

```html
<li title="A nice Content in the tree">
    <a href="#" <!-- ... -->>A nice Content in the tree</li>
    <ul class="ez-tree-level">
        <li title="A subitm of a nice Content in the tree">
        <!-- ... -->
        </li>
    </ul>
</li>
```

then the title is also associated with the subitems (the `ul`) of a given Location which is a bit weird semantically speaking. Also the `li` represents a Location in the tree while the `a` represents the Content item so the name (`title`) should stay at the Content item level since a Location has no name.